clib/memprof_coq.std.ml

```diff
@@ -0,0 +1,145 @@
+let is_interrupted _ = false [@@inline]
+
+module Resource_bind = struct
+  let ( let& ) f scope = f ~scope
+end
+
+module Mutex_aux = struct
+  external unlock: Mutex.t -> unit = "caml_mutex_unlock"
+
+  (* Critical sections :
+     - Mutex.lock does not poll on leaving the blocking section
+       since 4.12.
+     - Never inline, to avoid theoretically-possible reorderings with
+       flambda.
+     - Inline the call to Mutex.unlock to avoid polling in bytecode.
+       (workaround to the lack of masking)
+       mou*)
```

Comment:
mou?

Reply:
Ups! I was convinced this was on the original code, but indeed it is some cut & paste type.